Packs/CommonPlaybooks/Playbooks/playbook-Detonate_and_Analyze_File_-_Generic.yml

```diff
@@ -67,7 +67,12 @@ tasks:
                 value:
                   simple: '160: Windows 10,300: Linux Ubuntu 16.04,300: Linux Ubuntu 16.04'
       File:
-        simple: ${inputs.File}
+        complex:
+          root: inputs.File
+      EnableTor:
+        simple: "false"
+      EnvironmentID:
+        simple: '''160: Windows 10'''
```

Comment:
was this provided by default?

Reply:
Yes, it's the default value for the 'CrowdStrike Falcon Intelligence Sandbox Detonate and Analyze File' playbook.
![Screenshot 2023-11-14 at 21 11 15](https://github.com/demisto/content/assets/112805149/d3f24165-dabf-4636-9001-53f8bcc5b21f)